apps/extension/.env

```diff
@@ -1,6 +1,6 @@
 
 PRAX=lkpmkhpnhknhmibgnmmhdhgdilepfghe
-IDB_VERSION=32
+IDB_VERSION=33
```

Comment:
Needed for [this fix](https://github.com/penumbra-zone/web/pull/896/files#diff-8298d111bc94531ec943d5f024a9ab57657ad28670caea51e3aa9eda88bf3954)